### DISCUSSION TITLE
cant deleted "stars on github"

### DISCUSSION THREAD

**Discussion by unfairDude** [juliencrn/usehooks-ts#534] [Help]
im kinda a noob but I would like to know why deplyment fail when i try to remove the github widget about stars of this project

im editing page.tsx file and the code looks like this:

`</section> <section id="open-source" className="container py-8 md:py-12 lg:py-24"> <div className="mx-auto flex max-w-[58rem] flex-col items-center justify-center gap-4 text-center"> <h2 className="font-heading text-3xl leading-[1.1] sm:text-3xl md:text-6xl"> Proudly Open Source </h2> <p className="max-w-[85%] leading-normal text-muted-foreground sm:text-lg sm:leading-7"> usehooks-ts is open source and powered by open source software. </p> </div> </section> </> ) } `

**Comment by juliencrn** [comment]
Why would you delete this section?

**Comment by unfairDude** [comment]
Nvm It was issue on my end, it would be cool to add a file explaining how
to edit icons

**Comment by juliencrn** [comment]
I don't plan to do it, however, have you seen this: https://github.com/juliencrn/usehooks-ts/blob/master/.github/CONTRIBUTING.md
